src/github.com/mozilla-services/pushgo/simplepush/net.go

```diff
@@ -141,6 +170,12 @@ func (s *ServeCloser) close() error {
 	return nil
 }
 
+func (s *ServeCloser) hasConn(c net.Conn) bool {
+	s.connsLock.Lock()
+	defer s.connsLock.Unlock()
+	return s.conns[c]
```

Comment:
Huh, I had no idea you could return a map lookup like this to determine if the key is present.